cada/templates/about.html

```diff
@@ -0,0 +1,11 @@
c la <a href="https://cada.fr" target="_blank">CADA</a> afin de permettre la consultation en ligne des avis et des conseils émis depuis 1984. L’ensemble des données provient du jeu de données : <a href="http://www.data.gouv.fr/dataset/avis-et-conseils-de-la-cada" target="_blank">Avis et conseils de la CADA</a>, publié sur le site <a href="http://www.data.gouv.fr/" target="_blank">data.gouv.fr</a>.</p><h2 id="Le-cadre-légal">Le cadre légal</h2><p>La politique d’open data, dont le cadre légal a été profondémment renouvelé par les lois <a href="https://https://www.legifrance.gouv.fr/eli/loi/2015/12/28/PRMX1515110L/jo/texte" target="_blank">Gratuité et modalités de réutilisation des informations publiques</a> et <a href="https://https://www.legifrance.gouv.fr/affichTexte.do;jsessionid=70F06FAFE42AB53A449EF2AE222183BF.tplgfr23s_1?cidTexte=JORFTEXT000033202746&amp;categorieLien=id" target="_blank">République numérique</a> désormais codifiées dans le <a href="https://https://www.legifrance.gouv.fr/affichCode.do?cidTexte=LEGITEXT000031366350&amp;dateTexte=vig" target="_blank">Code des relations entre le public et d’administration</a>, puise en partie ses fondements dans le droit d’accès aux documents administratifs, institué par <a href="https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000339241" target="_blank">loi CADA en 1978</a>.</p><p>La loi CADA permet aux particuliers de demander des documents à l’ensemble des administrations centrales et locales. Ces dernières ont l’obligation d’y répondre, favorablement ou défavorablement.</p><h2 id="La-CADA">La CADA</h2><p>La Commission d’accès aux documents administratifs est une autorité administrative indépendante chargée de veiller à la liberté d’accès aux documents administratifs et aux archives publiques ainsi qu’à la réutilisation des informations publiques.</p><p>Elle peut être saisie par les personnes (physiques ou morales) qui se sont vues opposer une décision défavorable en matière d’accès aux documents administratifs ou de réutilisation des informations publiques. La commission peut aussi être saisie, à titre de conseil, par les administrations sollicitées en ces matières.</p><p>Le recours devant la CADA constitue un préalable obligatoire à tout recours contentieux (article L. 342-1 du CRPA).</p><p>Avant la saisine de la commission, une demande d’accès ou de publication de documents administratifs ou de réutilisation d’informations publiques doit être adressée à l’administration qui détient le document. En cas de refus, la saisine de la CADA est possible. Afin de faciliter les démanches des particuliers, la commission propose un <a href="https://https://www.cada.fr/formulaire-de-saisine" target="_blank">formulaire de saisine en ligne</a>.</p><p>Le site <a href="http://cada.data.gouv.fr" target="_blank">cada.data.gouv.fr</a> permet notamment de rechercher dans la doctrine de la CADA si cette dernière a déjà remis un avis sur un document.</p><h2 id="Liens-utiles">Liens utiles</h2><ul>
```

Comment:
Wow, get the job done but will be difficult to maintain.
We might need to do (later) to the same as [here](https://github.com/opendatateam/udata/blob/master/udata/core/site/views.py#L167-L178) (ie. extract the page content into an external markdown)